03-Mains/Heaving_Water_mass_transformation_decomposition.ipynb

```diff
@@ -0,0 +1,2108 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    plt.xlabel('Latitude ˚N',fontsize = 14)
</pre><p>Suggest units in parentheses: e.g. (°N).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>